### PR TITLE
Check if user registered

### PR DESCRIPTION
**Pull request by nazya08**

### CODE REVIEW

**Review by AlexGrig23 — APPROVED** [review]
LGFM

**Review by UsovDmytro — APPROVED** [review]

**Review by VasylKonon — APPROVED** [review]

**Review by vitaliileonov — APPROVED** [review]

File: src/main.py [hunk 1]
```diff
@@ -1,29 +1,43 @@
+import os
 from telegram import Update, InlineKeyboardButton, InlineKeyboardMarkup
 from telegram.ext import Application, CommandHandler, CallbackContext, CallbackQueryHandler
-import os
 from dotenv import load_dotenv
-load_dotenv()
```
Comment (vladysllav) [inline src/main.py]: Why do we move that row to bottom?

File: src/main.py [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
+from csv_file_handler import CSVFileHandler
+
 
 
 class FitnessBot:
 
     def __init__(self, token):
         self.app = Application.builder().token(token).build()
 
+        self.user_service = UserService(CSVFileHandler(os.getenv("PATH_TO_DB")))
```
Comment (vladysllav) [inline src/main.py]: Do not use CSV handler in this way, better initialize it inside UserService

**Review by vladysllav — CHANGES_REQUESTED** [review]
1. CSV handler should work with users file for users and other files for other purposes, so create another file for user storing. 
2. Use SCV handler inside UserService, dont pass handler as arg to UserService, initialize that handler inside __init__ method of UserService

**Pull request closed without merging**